telemetry/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.rdb
+software/data/
+software/provisioning/
```

Comment:
Add `__pycache__` and `.pyc` files

Reply:
We should use a standard gitignore, like the one [here](https://github.com/github/gitignore)